core/src/main/scala/org/http4s/ServerSentEvent.scala

```diff
@@ -24,13 +25,15 @@ import org.http4s.util.{Renderable, Writer}
 import scala.util.Try
 
 final case class ServerSentEvent(
-    data: String,
+    data: Option[String] = None,
     eventType: Option[String] = None,
     id: Option[EventId] = None,
-    retry: Option[Long] = None
+    retry: Option[Long] = None,
+    comments: Option[String] = None,
```

Comment:
Bikeshed: the plural name is weird to me.  It's just one comment, that might be split across multiple lines.

(`data` is wired into the spec, so I won't be a nerd and try to call that `datum`.)

Reply:
"comment" has been pushed